src/hotspot/share/metaprogramming/decay.hpp

```diff
@@ -26,16 +26,17 @@
 #define SHARE_METAPROGRAMMING_DECAY_HPP
 
 #include "memory/allStatic.hpp"
-#include "metaprogramming/removeCV.hpp"
 #include "metaprogramming/removeReference.hpp"
 
+#include <type_traits>
+
 // This trait trims the type from CV qualifiers and references.
 // This trait provides a subset of the functionality of std::decay;
 // array types and function types are not supported here.
 
 template <typename T>
 struct Decay: AllStatic {
-  typedef typename RemoveCV<typename RemoveReference<T>::type>::type type;
+  typedef typename std::remove_cv<typename RemoveReference<T>::type>::type type;
```

Comment:
Not that it really matters, since this code is going away shortly too, but:
`using type = std::remove_cv_t<typename RemoveReference<T>::type>;`
That is, use the `_t` suffixed traits when available.  And prefer type aliases now that we have them.

Reply:
Done.